wordpress-mobile/WordPress-Login-Flow-Android
# Merge WooCommerce Dark Mode changes

This PR merges in the changes from the WooCommerce dark mode project contained in these two PRs:
- [Dark Mode: Base login screens](https://github.com/woocommerce/woocommerce-android/pull/2266)
- [Dark Mode v2: Site address help](https://github.com/woocommerce/woocommerce-android/pull/2330)

As well as this older unrelated PR that must've been missed for merging back to this repo:
* [Gradle version upgraded to 3.5.3](https://github.com/woocommerce/woocommerce-android/pull/1622)
* [Fixed bug when the forgot password url in username password screen was not valid](https://github.com/woocommerce/woocommerce-android/pull/1589)
* [Added logic to fetch the correct SiteModel from the local db for the incoming url](https://github.com/woocommerce/woocommerce-android/pull/1756)

## To Test
I've opened [a PR in WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/pull/12010) to ensure these changes doe not conflict with that app. Please visit that PR to test. 

** Only one developer required to review and approve this PR
